web/src/frontend/src/digisos/skjema/inntektFormue/verdier/Verdier.tsx

```diff
@@ -0,0 +1,120 @@
+import * as React from 'react';
+import {
+    connectSoknadsdataContainer,
+    SoknadsdataContainerProps
+} from "../../../../nav-soknad/redux/soknadsdata/soknadsdataContainerUtils";
+import {FormattedHTMLMessage, InjectedIntlProps, injectIntl} from "react-intl";
+import {SoknadsSti} from "../../../../nav-soknad/redux/soknadsdata/soknadsdataReducer";
+import Sporsmal, {LegendTittleStyle} from "../../../../nav-soknad/components/sporsmal/Sporsmal";
+import {getFaktumSporsmalTekst} from "../../../../nav-soknad/utils";
+import JaNeiSporsmal from "../../../../nav-soknad/faktum/JaNeiSporsmal";
+import {Verdier} from "./VerdierTypes";
+import CheckboxPanel from "../../../../nav-soknad/faktum/CheckboxPanel";
+import TextareaEnhanced from "../../../../nav-soknad/faktum/TextareaEnhanced";
+import NivaTreSkjema from "../../../../nav-soknad/components/nivaTreSkjema";
+
+const MAX_CHARS = 500;
+const Verdier = "inntekt.eierandeler";
+
+type Props = SoknadsdataContainerProps & InjectedIntlProps;
+
+export class VerdierView extends React.Component<Props, {}> {
+
+    componentDidMount(): void {
+        this.props.hentSoknadsdata(this.props.brukerBehandlingId, SoknadsSti.VERDIER);
+    }
+
+    handleClickJaNeiSpsm(verdi: boolean) {
+        const {brukerBehandlingId, soknadsdata} = this.props;
+        const verdier: Verdier = soknadsdata.inntekt.verdier;
+        verdier.bekreftelse = verdi;
+        if (!verdi) {
+            verdier.bolig = false;
+            verdier.campingvogn = false;
+            verdier.kjoretoy = false;
+            verdier.fritidseiendom = false;
+            verdier.annet = false;
+            verdier.beskrivelseAvAnnet = "";
+        }
+        this.props.oppdaterSoknadsdataSti(SoknadsSti.VERDIER, verdier);
+        this.props.lagreSoknadsdata(brukerBehandlingId, SoknadsSti.VERDIER, verdier);
+    }
+
+    handleClickRadio(idToToggle: string) {
+        const {brukerBehandlingId, soknadsdata} = this.props;
+        const verdier: Verdier = soknadsdata.inntekt.verdier;
+        verdier[idToToggle] = !verdier[idToToggle];
+        this.props.oppdaterSoknadsdataSti(SoknadsSti.VERDIER, verdier);
+        this.props.lagreSoknadsdata(brukerBehandlingId, SoknadsSti.VERDIER, verdier);
+    }
+
+    onChangeAnnet(value: string) {
+        const {soknadsdata} = this.props;
+        const verdier: Verdier = soknadsdata.inntekt.verdier;
+        verdier.beskrivelseAvAnnet = value;
+        this.props.oppdaterSoknadsdataSti(SoknadsSti.VERDIER, verdier);
+    }
+
+    onBlurTekstfeltAnnet() {
+        const {brukerBehandlingId, soknadsdata} = this.props;
+        const verdier: Verdier = soknadsdata.inntekt.verdier;
+        this.props.oppdaterSoknadsdataSti(SoknadsSti.VERDIER, verdier);
```

Comment:
Linjen `this.props.oppdaterSoknadsdataSti(SoknadsSti.VERDIER, verdier);` er overflødig. Verdien er allerede satt på redux store av OnChangeAnnet().